README.md

```diff
@@ -83,15 +83,15 @@ version 1.13.0 or later.
 ```sh
   -b           optional  Do not print colors
   -h           optional  Print this help message
-  -l FILE      optional  Log output in FILE
+  -l FILE      optional  Log output in FILE inside docker bench container
   -c CHECK     optional  Comma delimited list of specific check(s)
   -e CHECK     optional  Comma delimited list of specific check(s) to exclude
   -i INCLUDE   optional  Comma delimited list of patterns within a container or image name to check
   -x EXCLUDE   optional  Comma delimited list of patterns within a container or image name to exclude from check
 ```
 
 By default the Docker Bench for Security script will run all available CIS tests
-and produce logs in the current directory named `docker-bench-security.sh.log.json`
+and produce logs in the current directory inside the container, named `docker-bench-security.sh.log.json`
```

Comment:
Instead of adding this, maybe add a note beneath saying that if the container is used, the logs are stored in the container unless a volume is mounted.